### DISCUSSION TITLE
Can I use yarn to install the packages?

### DISCUSSION THREAD

**Discussion by tiago138** [mason-org/mason.nvim#1305] [Q&A]
There is a way to use yarn instead of npm to install the packages?

**Comment by williamboman** [comment]
There's no way to use yarn in place of npm. I'm curious as to why you'd want to do that?

**Reply by tiago138** [reply]
This week npm is just saying I am having proxy problems and don't install anything, but yarn is working.

**Reply by williamboman** [reply]
You might want to check your global npm settings (`~/.npmrc`). You could also try clearing the proxy settings (you might want to make a backup of `~/.npmrc` before doing this):

```
$ npm config rm proxy
$ npm config rm https-proxy
```

**Reply by tiago138** [reply]
Thanks, it seems to be working fine now.
